tests/ui/pin_project/add-attr-to-struct.stderr

```diff
@@ -1,4 +1,4 @@
-error: duplicate #[pin] attribute
+error: expected attribute arguments in parentheses: `pin(...)`
```

Comment:
I adjusted the test to use an explicit `#[pin]` rather than your helper macro. Duplicate `#[pin]` still yields the correct error, it's a macro ordering issue that yields the must-have-argument issue.